addon/PlacesProvider.js

```diff
@@ -782,7 +782,6 @@ Links.prototype = {
                     FROM moz_bookmarks b, moz_places p
                     WHERE type = :type
                     AND b.fk = p.id
-                    AND p.last_visit_date IS NOT NULL
```

Comment:
Looks like `Bookmarks.jsm` avoids `place:` rows by using `hash()`:
https://dxr.mozilla.org/mozilla-central/source/toolkit/components/places/Bookmarks.jsm#1503-1504

`AND url_hash NOT BETWEEN hash("place", "prefix_lo") AND hash("place", "prefix_hi")`